03/app.js

```diff
@@ -0,0 +1,13 @@
+
+const buttonEl = document.querySelectorAll("button");
+
+const changeText = function () {
+    this.innerText = "clicked";
+    console.log("clicked");
```

Comment:
Zwróć uwagę, że teraz to rozwiązanie działa w taki sposób, że każde kliknięcie w button powoduje wyświetlenie tekstu w konsoli. W zadaniu było napisane, że tylko pierwsze kliknięcie tak zadziała. Postaraj się to poprawić :)